### ISSUE TITLE
USWDS - Link: Update placement and allow for opt out

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds#5942]
Place label after link to follow W3C recommendation. Allow developers who have a custom implementation the option to easily remove label .

_Originally suggested by @aduth in https://github.com/uswds/uswds/issues/5166#issuecomment-2137461653_

**Comment by brunerae** [comment]
Passing this to the A11Y team (@amycole501 / @alex-hull ) to confirm if this is something you all think makes sense to pursue. Your action items:

- [ ] Confirm if this ask is logical/something we want to pursue and leave a comment with your thoughts
- [ ] Add a value to the `weight` field indicating how important this work would be/level of priority

After you've done the above, please update the Issue by:

**If we want to pursue the work**
- [ ] Removing `Needs: Confirmation` label
- [ ] Changing `Active Role` field from `A11Y` to `Dev`
- [ ] Changing `Status` field from `Needs refinement` to `Ready to schedule`

**If don't want to pursue the work**
- [ ] Removing `Needs: Confirmation` label
- [ ] Changing `Active Role` field from `A11Y` to `Fed`
- [ ] Add label `Status: will not pursue`
- [ ] Changing `Status` field from `Needs refinement` to `Fed Final Review`

Thank you!

**Comment by amycole501** [comment]
I agree that the "external link" or warning of opening in a new window needs to come after the link is announced to improve clarity. While the issue isn't a blocker for users, it is at least possibly confusing. Moving the announcement to the end of the link announcement will sound more like "Visit our website link name (link opens in a new window") which makes more sense to English speakers and users of English assistive tech.

**Comment by aduth** [comment]
>Allow developers who have a custom implementation the option to easily remove label .

To support this, and continuing my original comment's context of our needing to support multilingual labels, I wonder if the existing theme option could be expanded to support different value types, such as:

- A string to override the default label (currently supported)
   - Example: `$theme-external-link-sr-label-tab-same: "opens new tab"`
- A `false` value to disable the screen reader label
   - Example: `$theme-external-link-sr-label-tab-same: false`
- A [map](https://sass-lang.com/documentation/values/maps/) of `locale` ➡️ `label` values, applied with `[lang]` selector
   - Example: `$theme-external-link-sr-label-tab-same: ("en": "opens new tab", "es": "abrir nueva pestaña")`

Here's an example implementation: https://github.com/uswds/uswds/compare/b5a332a...aduth:aduth-external-link-multilingual

(Not opening a pull request yet since it doesn't resolve the label placement after link, which is more difficult to solve)

**Comment by chandracarney** [comment]
In June, we discussed that this needs a quick further look. Here are our initial thoughts:

- Needs research / verification / investigation by someone with accessibility expertise.
- Possible light engineering to make the update, and a light review from another engineer.
